homeassistant/components/lg_thinq/__init__.py

```diff
@@ -55,29 +58,22 @@ async def async_setup_coordinators(
     thinq_api: ThinQApi,
 ) -> None:
     """Set up coordinators and register devices."""
-    entry.runtime_data = {}
-
-    # Get a device list from the server.
+    # Get a list of ha bridge.
     try:
-        device_list = await thinq_api.async_get_device_list()
+        ha_bridge_list = await async_get_ha_bridge_list(thinq_api)
```

Comment:
what's the `ha_bridge`?

Reply:
Should we just name it `bridge_list` here? It's strange to call it ha_bridge since we're already in HA

Reply:
Renamed